docs/changelog.md

```diff
@@ -1,3 +1,11 @@
+## 0.17.1 (Feb 9, 2021)
+
+### Added
+
+* Add handler option for bubbles and keyboard buttons (`handler`). If `client`,
```

Comment:
Add \`handler\` option for bubbles and keyboard buttons (`handler`). If `client`,